### PR TITLE
chore: update README and TypeScript error in Annotation storybook

### PR DESCRIPTION
**Pull request by TCL735**
Addresses several issues in #422 

Closes issues **a, d, f, g, h, i**

### CODE REVIEW

File: giraffe/README.md [hunk 1]
```diff
@@ -11,157 +11,186 @@ A React-based visualization library powering the data visualizations in [InfluxD
...
+       }
+     <script>
+   ```
 
 1. And have React render that element. Change the `ReactDOM.render` call to:
-    ```js
-    ReactDOM.render(
-      React.createElement(PlotRenderer, null, null),
-      document.getElementById('root')
-    );
```
Comment (hoorayimhelping) [inline giraffe/README.md:77]: i'd prefer if you didn't remove semicolons - those were put in deliberately. i know we don't use them in our conventions, but they're required in JavaScript expressions, and in an html quickstart that's copypasta-ed into an indexl.html file, they won't be built and inserted into the output. which means we'd be at the mercy of the compiler guessing our intentions.

File: giraffe/README.md [hunk 2]
```diff
@@ -11,157 +11,186 @@ A React-based visualization library powering the data visualizations in [InfluxD
...
+
+   Open up `index.html`. You should see Giraffe Plot Goes Here where it used to say Hello World.
 
 1. Our last step is to create some fake data and then draw that fake data into a plot. First, we'll create a configuration object that tells Giraffe to render a line graph. In the render method of `PlotRenderer`, add the following code:
-    ```js
-    const lineLayer = {
-      type: "line",
-      x: "_time",
-      y: "_value"
-    };
```
Comment (hoorayimhelping) [inline giraffe/README.md:87]: semicolon

File: giraffe/README.md [hunk 3]
```diff
@@ -11,157 +11,186 @@ A React-based visualization library powering the data visualizations in [InfluxD
...
+     x: '_time',
+     y: '_value',
+   }
+   ```
 
 1. Let's create the fake data. After the `lineLayer` code, add the following:
-    ```js
-    const table = Giraffe.newTable(3)
-      .addColumn('_time', 'dateTime:RFC3339', 'time', [1589838401244, 1589838461244, 1589838521244])
-      .addColumn('_value', 'double', 'number', [2.58, 7.11, 4.79]);
```
Comment (hoorayimhelping) [inline giraffe/README.md:94]: semicolon

File: giraffe/README.md [hunk 4]
```diff
@@ -11,157 +11,186 @@ A React-based visualization library powering the data visualizations in [InfluxD
...
+     ])
+     .addColumn('_value', 'double', 'number', [2.58, 7.11, 4.79])
+   ```
 
 1. Now we'll combine them into a `config` object. Add this after the line that creates `table`:
-    ```js
-    const config = {
-      table,
-      layers: [lineLayer]
-    };
```
Comment (hoorayimhelping) [inline giraffe/README.md:102]: semicolon

File: giraffe/README.md [hunk 5]
```diff
@@ -11,157 +11,186 @@ A React-based visualization library powering the data visualizations in [InfluxD
...
+   const config = {
+     table,
+     layers: [lineLayer],
+   }
+   ```
 
 1. Finally, let's create a `Plot` with this configuration and render it. Below the line that creates `config` in the `render` method, add the following code:
-    ```js
-    const SimplePlot = React.createElement(Giraffe.Plot, {config}, null);
-    return React.createElement('div', {style}, SimplePlot);
```
Comment (hoorayimhelping) [inline giraffe/README.md:108]: semicolon x2

File: giraffe/README.md [hunk 6]
```diff
@@ -11,157 +11,186 @@ A React-based visualization library powering the data visualizations in [InfluxD
...
-    <body id="home">
-      <main id="root"></main>
-      <script type="text/javascript">
-        class PlotRenderer extends React.Component {
-          render() {
-            const style = {
-              width: "calc(70vw - 20px)",
-              height: "calc(70vh - 20px)",
-              margin: "40px",
-            };
```
Comment (hoorayimhelping) [inline giraffe/README.md:133]: semicolon

File: giraffe/README.md [hunk 7]
```diff
@@ -11,157 +11,186 @@ A React-based visualization library powering the data visualizations in [InfluxD
...
-
-            const lineLayer = {
-              type: "line",
-              x: "_time",
-              y: "_value"
-            };
```
Comment (hoorayimhelping) [inline giraffe/README.md:139]: semicolon

File: giraffe/README.md [hunk 8]
```diff
@@ -11,157 +11,186 @@ A React-based visualization library powering the data visualizations in [InfluxD
...
-
-            const table = Giraffe.newTable(3)
-              .addColumn('_time', 'dateTime:RFC3339', 'time', [1589838401244, 1589838461244, 1589838521244])
-              .addColumn('_value', 'double', 'number', [2.58, 7.11, 4.79]);
```
Comment (hoorayimhelping) [inline giraffe/README.md:143]: semicolon

File: giraffe/README.md [hunk 9]
```diff
@@ -11,157 +11,186 @@ A React-based visualization library powering the data visualizations in [InfluxD
...
-
-            const config = {
-              table,
-              layers: [lineLayer]
-            };
```
Comment (hoorayimhelping) [inline giraffe/README.md:148]: semicolon

File: giraffe/README.md [hunk 10]
```diff
@@ -11,157 +11,186 @@ A React-based visualization library powering the data visualizations in [InfluxD
...
-
-            const SimplePlot = React.createElement(Giraffe.Plot, {config}, null);
-            return React.createElement('div', {style}, SimplePlot);
-          }
-        }
-
-        ReactDOM.render(
-          React.createElement(PlotRenderer),
-          document.getElementById('root')
-        );
```
Comment (hoorayimhelping) [inline giraffe/README.md:158]: semicolon

**Review by hoorayimhelping — APPROVED** [review]
other than the semicolon issue, looks good to me!

**Comment by TCL735** [comment]
@hoorayimhelping whoops, the semicolon issue is an auto-format from Prettier. Which leads me to believe that the indentation is not worth addressing at all. I don't really care about the indentation except that it is auto-formatted by Prettier. But since Prettier is also removing something we want (semicolons), it is better just to disable the extension for html files, i guess.

I will revert the formatting & semicolon changes.

**Comment by hoorayimhelping** [comment]
lousy machines, overstepping their bounds

**Review by hoorayimhelping — APPROVED** [review]

**Pull request merged**